ckan/logic/action/get.py

```diff
@@ -1479,7 +1479,7 @@ def user_show(context, data_dict):
     if asbool(data_dict.get('include_datasets', False)):
         user_dict['datasets'] = []
 
-        fq = "+creator_user_id:{0}".format(user_dict['id'])
+        fq = "creator_user_id:{0}".format(user_dict['id'])
```

Comment:
There is no sense in ommitting `+` as we are searching for datasets of this particular user

Reply:
The `+` at the start of the query was a workaround for the fact that because of the error at https://github.com/ckan/ckan/pull/4962#discussion-diff-325791829L1887, whatever you put into the `fq` here would be ignored. With the change there, the `+` is no longer needed (and in fact causes a malformed query with `++`).

Reply:
Plus sign is just a signal for Solr that this filter `MUST` match occurrence. If it results in malformed query, we should look at the place where code assumes that there is no `+` in `fq` and adds second `+`. This line itself doesn't contain any contradiction and this change IS implicit fix of but that IS NOT created by this line.